skbio/stats/tests/test_power.py

```diff
@@ -477,18 +477,23 @@ def test__identify_sample_groups(self):
 
     def test__identify_sample_groups_not_strict(self):
         # Defines the know values
-        known_pairs = {0: [['PP'], ['CD', 'NR']],
-                       1: [['MM', 'WM'], ['MH']],
-                       2: [['GW'], ['CB']]}
+        known_pairs = {1: [np.array(['PP'], dtype=object),
+                           np.array(['CD', 'NR'], dtype=object)],
+                       0: [np.array(['MM', 'WM'], dtype=object),
```

Comment:
The 0 and 1 are proxies for a group description. The change in order reflects a change in the order returned by groupby (again, I think this is a pandas update result). The groups have remained consistent.